src/components/AllDrinksGrid/AllDrinksGrid.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import './AllDrinksGrid.css'
+import mixedDrinkRecipes from '../../mock-data' // we will update with a real API call
+import NavBar from '../NavBar/NavBar'
+
+function AllDrinksGrid() {
+
+ return (
+    <div>
+      <section className="home-title">
+        <h1>Mix Master</h1>
+      </section>
+        <NavBar />
+      <section className="drinks-grid">
+        {mixedDrinkRecipes.map((recipe) => {
```

Comment:
Might be worth destructuring `recipe` here so you don't have to keep referencing `recipe.` in the next several lines of code.